lib/trie/trie.go

```diff
@@ -18,10 +21,9 @@ var EmptyHash, _ = NewEmptyTrie().Hash()
 // Use NewTrie to create a trie that sits on top of a database.
 type Trie struct {
 	generation  uint64
-	root        node
+	root        Node
 	childTries  map[common.Hash]*Trie // Used to store the child tries.
 	deletedKeys []common.Hash
-	parallel    bool
```

Comment:
Because it was always true in production code.
And it being true might use a lot of stack memory with the current parallel hashing implementation.

This is improved/fixed in #2081 where opportunistic hashing is always enabled.